de4a-kafka-client/src/main/java/eu/de4a/kafkaclient/DE4AKafkaManager.java

```diff
@@ -160,4 +192,36 @@ public static void shutdown ()
     final ProducerRecord <String, String> aMessage = new ProducerRecord <> (DE4AKafkaSettings.getKafkaTopic (), sKey, sValue);
     return getOrCreateProducer ().send (aMessage, aKafkaCallback);
   }
+
+  @Nonnull
+  public static void send(@Nullable final String sKey,
+                          @Nonnull final String sValue) {
+
+    ValueEnforcer.notNull(sValue, "Value");
+    HttpURLConnection conn;
+    int code = -1;
+    try {
+        conn = createUrlConnection();
+        String data = "{\"records\": [{\"key\": \"" + sKey + "\", \"value\": \""+ sValue +"\"}]}";
```

Comment:
And I suggest to build a structured JSON e.g. with (untedted)
```
new JsonObject ().add ("records", new JsonArray ().add (new JsonObject ().add ("key", sKey).add ("value", sValue))).getAsJsonString ();
```
It does the quoting correct etc.

Reply:
Is this JsonObject/Array an already included dependency or should I add the javax.json one?

Reply:
you may want to use com.helger.com:ph-json - it is used in other places as well